block-node/app/docker/update-env.sh

```diff
@@ -36,9 +30,15 @@ else
   echo "JAVA_OPTS='-Xms16G -Xmx16G'" >> .env
 fi
 
-# file is mounted in the docker-compose.yml, changes to the file will be reflected in the container by simply restarting it
-echo "JAVA_TOOL_OPTIONS='-Djava.util.logging.config.file=/app/logs/config/logging.properties'" >> .env
-
+if [ true = "$is_debug" ]; then
+  # The server will wait for the debugger to attach on port 5005
+  # JProfiler can attach on port 8849
+  echo "JAVA_TOOL_OPTIONS='-Djava.util.logging.config.file=/app/logs/config/logging.properties -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005' -agentpath:/path/to/libjprofilerti.so=port=8849 " >> .env
```

Comment:
I think we need to remove the below `'` as they will result in having an environment variable `-agentpath:/path/to/libjprofilerti.so=port=8849` which is not correct (they conflict with the `"`s). Moreover, maybe we can completely remove this part `-agentpath:/path/to/libjprofilerti.so=port=8849` not really sure we need this.
```suggestion
  echo "JAVA_TOOL_OPTIONS=-Djava.util.logging.config.file=/app/logs/config/logging.properties -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005 -agentpath:/path/to/libjprofilerti.so=port=8849" >> .env
```

Reply:
That part is used for Profiling, should only be on debug mode.

Thanks for noticing, somehow debugging remotely on the container is working, but probably the profiling is not, never tested that to be honest.

Have you tried this with your suggested changes?

Reply:
I noticed the wrong env var when I made changes to the simulator java opts, that one was a copy paste. This will produce the same result. I know about the profiling stuff, I think it never worked (I mean the option, profiling did work if I remember correctly). This is why I think we can omit the profiling opt (could be wrong tho, need to double check).

Reply:
P.S. debugging is always working because the debugging part is correct even with the `'`s.

Reply:
But do try it out though.